ui/src/app/base/components/TableMenu/TableMenu.tsx

```diff
@@ -20,11 +21,13 @@ const TableMenu = <L,>({
   onToggleMenu,
   position = "left",
   positionNode,
+  "aria-label": ariaLabel,
```

Comment:
This could spread additional props on `ContextualMenu` rather than defining specific props.